util/BUILD

```diff
@@ -16,6 +16,13 @@ genrule(
     stamp = 1,  # this provides volatile-status.txt
 )
 
+genrule(
+    name = "scm_revision_file",
+    outs = ["scm_revision.txt"],
+    cmd = """awk '/BUILD_SCM_REVISION/ { print $$2 }' bazel-out/volatile-status.txt > $@""",
```

Comment:
I don't speak `awk` but I trust you :) Curious, why not `sed`?

Reply:
Hehe, I actually just ripped off of ":ot_version_file" from the same BUILD file.

If you're curious, the awk command is just saying "for lines matching BUILD_SCM_REVISION, print column 2."